content/03.when-to-use-dl.md

```diff
@@ -1,51 +1,45 @@
+In the areas of biology where data collection is thoroughly automated, such as DNA sequencing, large amounts of high quality data may in fact be available.
+However, areas of biology that rely on manual data collection may not possess enough data to effectively train and apply deep learning models.
+Though there are methods that try to increase the amount of training data, such as data augmentation (in which existing data is slightly manipulated in an attempt to yield "new" samples) and weak supervision (in which simple labeling heuristics are combined to produce noisy, probabilistic labels) [@arxiv:1605.07723v3], these methods cannot overcome a complete shortage of data.
+A general rule of thumb is that deep learning can be considered for datasets with at least one hundred samples per class [@arxiv:1511.06348], although it is really best suited for use with datasets that contain orders of magnitude more samples.
 
-Furthermore, training deep learning models can be very demanding, often requiring extensive computing infrastructure and patience to achieve state-of-the-art performance [@doi:10.1109/JPROC.2017.2761740].
-In some deep learning contexts, such as generating human-like text, state-of-the-art models have over one hundred billion parameters [@arxiv:2005.14165].
-Training such large models from scratch can be a costly and time-consuming undertaking [@arxiv:1906.02243].
-Luckily, most deep learning research in biology will not require nearly as much computation, though it usually requires more than can be done feasibly on an individual consumer-grade device.
-Specialized hardware such as discrete graphics processing units (GPUs) or custom deep learning accelerators can dramatically reduce the time and cost required to train models, but this hardware is not universally accessible.
-Currently, both GPU- and deep learning-optimized accelerator-equipped servers can be rented from cloud providers, though working with these servers adds additional cost and complexity.
-As deep learning becomes more popular, these accelerators are likely to be more broadly available (for example, recent-generation iPhones already have such hardware).
-In contrast, traditional machine learning training can often be done on a laptop (or even a \$5 computer [@arxiv:1809.00238]) in seconds to minutes.
+Training deep learning models is also very demanding, and often requires extensive computing infrastructure and patience to achieve state-of-the-art performance [@doi:10.1109/JPROC.2017.2761740].
+In some deep learning contexts, such as generating human-like text, state-of-the-art models have over one hundred billion parameters [@arxiv:2005.14165] and require very costly and time-consuming training [@arxiv:1906.02243].
+Fortunately, most deep learning applications in biology do not require nearly this much training, although they usually do require computational resources beyond those available on typical consumer-grade devices.
```

Comment:
```suggestion
Though deep learning applications in biology rarely require this much training, they can require computational resources beyond those available on typical consumer-grade devices.
```
Is a discrete GPU really "beyond those available on typical consumer-grade devices"? I went with it but I'm not sure this is really accurate looking at the current literature.

Reply:
Not quite "beyond" it but not super common. How about:

```suggestion
Though deep learning applications in biology rarely require this much training, they can require computational resources beyond those available on consumer-grade devices such as laptops or office desktops.
```

Gaming PCs all have discrete GPUs but your average office PC probably wouldn't have one.